multinode/bootstrapping.md

```diff
@@ -0,0 +1,191 @@
+`bootstrap` is false. IMHO, we should add that. /Mats]
+
+### Validation of the database
+
+If `bootstrap` is false, the database will be validated by checking
+that the encoding, character type, and collation are the same as the
+current database on the access node.
+
+## Setting up the extension
+
+The `timescaledb` extension needs to be installed in the database on
+the data node, so the next step is ensuring that there is a valid
+extension installed in the database by either: Creating the extension,
+or verifying that the extension is compatible with the extension
+installed on the access node.
+
+### Creating the schema for the extension
+
+To install the extension, a schema is created to store the objects
+that the `timescaledb` extension needs. The schema name is decided
+looking what schema is used for the extension on the access node and
+using the same name on the data node. If you have created the
+`timescaledb` extension in the default manner, `public` will be used
+for the schema on the access node, hence `public` will be used on the
+data node as well.
+
+If the schema name for the extension on the access node is `public`,
+no new schema is created, but otherwise it is expected that the schema
+name does *not* exist. In the event that the schema name exists (but
+is not `public`), an error will be raised the bootstrapping aborted.
+
+Alternative description:
+> If the schema already exists on the data node the access node will
+> verify that it is correctly configured and proceed with installing
+> the extension.
+
+To change the schema used on the access node and data node, you can
+create a new schema, then create the extension with the schema that
+you want:
+
+```
+CREATE SCHEMA magic;
+CREATE EXTENSION timescaledb SCHEMA magic;
+```
+
+When validating the schema, it is necessary that the schema is owned
+by the current user, or is `public`.
+
+> :TIP: This will place all the Timescale functions (for example,
+> `create_hypertable` in the `magic` schema), so you need to qualify
+> all function references with `magic`. For example:
+>
+> ```
+> SELECT * FROM magic.create_hypertable('conditions', 'time', 3);
+> ```
+
+### Installing the extension
+
+After the schema is created, the extension is installed using the
+`CREATE EXTENSION` command with the `SCHEMA` option to refer to the
+schema from the section above. The schema name for the extension will
+be either `public` or a dedicated schema name.
+
+If the extension already exists in the database, an error will be
+raised and the bootstrapping aborted.
+
+Proposed changes:
+> If the extension already exists, it is validated to ensure that it
+> can be used with the access node and the procedure proceeds to set
+> the metadata.
+
+### Validating the extension
+
+Validation of the extension checks that the extension on the data node
+is of a version that is compatible with the version on the access node
+and that the access node have permissions to use the extension.
```

Comment:
Do you mean that the user has permission to use the extension?

Reply:
I suggest to remove this introduction, since the explanation is close enough.

Reply:
```suggestion
and that the access node has permissions to use the extension.
```